extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -60,6 +60,21 @@ public static function getSvg( $name, $classNames = '', $alt = '' ) {
 		}
 	}
 
+	public static function getApiImage( $model, $classNames = '', $alt = '' ) {
+		if ( $model['type'] === 'wds-svg' ) {
+			return static::getSvg( $model['name'], $classNames, $alt );
+		} elseif ($model['type'] === 'image-external') {
+			return Html::element( 'img', [
```

Comment:
Using Html deepens coupling with MW class library and reduces potential for treating this as part of DS integration library. Has simplexml been considered and if not, why?